packages/components/src/components/table-stateless/component.tsx

```diff
@@ -715,77 +736,80 @@ export class KolTableStateless implements TableStatelessAPI {
-						 * The following element allows the table to receive focus without providing redundant content to screen readers.
-						 * The `div` is technically not allowed here. But any allowed element would mutate the table semantics. Additionally, the `&nbsp;` is necessary to
-						 * prevent screen readers from just reading "blank".
-						 */}
-						{/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
-						<div class="focus-element" tabindex={this.tableDivElementHasScrollbar ? '0' : undefined} aria-describedby="caption">
-							&nbsp;
-						</div>
-
-						<caption id="caption">{this.state._label}</caption>
-
-						{Array.isArray(this.state._headerCells.horizontal) && (
-							<thead>
-								{[
-									this.state._headerCells.horizontal.map((cols, rowIndex) => (
-										<tr key={`thead-${rowIndex}`}>
-											{this.state._selection && this.renderHeadingSelectionCell()}
-											{rowIndex === 0 && this.renderHeaderTdCell()}
-											{Array.isArray(cols) &&
-												cols.map((cell, colIndex) => {
-													if (cell.asTd === true) {
-														return (
-															<td
-																key={`thead-${rowIndex}-${colIndex}-${cell.label}`}
-																class={{
-																	[cell.textAlign as string]: typeof cell.textAlign === 'string' && cell.textAlign.length > 0,
-																}}
-																colSpan={cell.colSpan}
-																rowSpan={cell.rowSpan}
-																style={{
-																	textAlign: cell.textAlign,
-																	width: cell.width,
-																}}
-																ref={
-																	typeof cell.render === 'function'
-																		? (el) => {
-																				this.cellRender(cell, el);
-																			}
-																		: undefined
-																}
-															>
-																{typeof cell.render !== 'function' ? cell.label : ''}
-															</td>
-														);
-													} else {
-														return this.renderHeadingCell(cell, rowIndex, colIndex, false);
-													}
-												})}
-										</tr>
-									)),
-									this.renderSpacer('head', this.state._headerCells.horizontal),
-								]}
-							</thead>
-						)}
-						<tbody>{dataField.map((row: (KoliBriTableCell & KoliBriTableDataType)[], rowIndex: number) => this.renderTableRow(row, rowIndex, true))}</tbody>
-						{this.renderFoot()}
-					</table>
-				</div>
-			</Host>
+			/* Firefox automatically makes the following div focusable when it has a scrollbar. We implement a similar behavior cross-browser by allowing the
+			 * <div class="focus-element"> to receive focus. Hence, we disable focus for the div to avoid having two focusable elements by setting `tabindex="-1"`
+			 */
+			/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */
+			<div ref={(element) => (this.tableDivElement = element)} class="kol-table" tabindex={this.tableDivElementHasScrollbar ? '-1' : undefined}>
+				<table
+					class="kol-table__container"
```

Comment:
```suggestion
					class="kol-table__table"
```
Fände ich hier passender